addons/mrp_subcontracting/models/stock_picking.py

```diff
@@ -57,7 +57,7 @@ def _action_done(self):
             if not production:
                 continue
             if len(production) > 1:
-                raise UserError("It shouldn't happen to have multiple production to record for the same subcontracted move")
+                raise UserError(_("Multiple production should not be recorded for the same subcontracted move."))
```

Comment:
```suggestion
                raise UserError(_("There shouldn't be multiple productions to record for the same subcontracted move."))
```

I think there might also be a copy paste error in your corresponding pot file change too.

Reply:
@ticodoo thanks and updated